src/nl/surf/eduhub_rio_mapper/api.clj

```diff
@@ -85,7 +90,17 @@
             "education-specifications" "education-specification"
             "programs"                 "program"})
 
-(def actions #{"upsert" "delete" "dry-run-upsert" "link"})
+(def actions #{"upsert" "delete" #_"delete-by-code" "dry-run-upsert" "link"})
+
+(defn action-id-for [command id type]
+  (if (= "delete-by-code" command)
+    (let [name-id (if (= type "education-specification")
+                    ::rio/opleidingscode
+                    ::rio/aangeboden-opleiding-code)]
+      {:action "delete"
+       name-id id})
+    {:action    command
+     ::ooapi/id id}))
```

Comment:
This should go into the `cli` ns because the `api` does not do `delete-by-code`.